### PR TITLE
Cli dev

### PR DESCRIPTION
**Pull request by a5anka**

### CODE REVIEW

File: cmd/cmd/serverAdd.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+		printServerAddHelp()
+	}
+}
+
+func executeServerAddCmd(args []string) {
+	var result = utils.RemoteConfigData.AddRemote(args[0], args[1], args[2])
+	if result != nil {
+		utils.HandleErrorAndExit("Error: ", result)
+	}
+	utils.RemoteConfigData.Persist(utils.GetServerConfigFilePath())
```
Comment (a5anka) [inline cmd/cmd/serverAdd.go]:
```
Found a 8 line (44 tokens) duplication in the following files: 
Starting at line 73 of /home/asanka/repos/wso2/micro-integrator/cmd/cmd/serverAdd.go
Starting at line 73 of /home/asanka/repos/wso2/micro-integrator/cmd/cmd/serverUpdate.go
```
```go
	var result = utils.RemoteConfigData.AddRemote(args[0], args[1], args[2])
	if result != nil {
		utils.HandleErrorAndExit("Error: ", result)
	}
	utils.RemoteConfigData.Persist(utils.GetServerConfigFilePath())
}
```

File: cmd/utils/remoteConfig.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+	return nil
+}
+
+func (remoteConfig *RemoteConfig) UpdateRemote(name string, host string, port string) error {
+
+	remotes := &RemoteConfigData.Remotes
+	if _, exists := (*remotes)[name]; !exists {
+		return errors.New("no such remote")
+	}
```
Comment (a5anka) [inline cmd/utils/remoteConfig.go]: Found a 4 line (31 tokens) duplication in the following files: 
Starting at line 49 of /home/asanka/repos/wso2/micro-integrator/cmd/utils/remoteConfig.go
Starting at line 65 of /home/asanka/repos/wso2/micro-integrator/cmd/utils/remoteConfig.go
Starting at line 81 of /home/asanka/repos/wso2/micro-integrator/cmd/utils/remoteConfig.go
```go
func (remoteConfig *RemoteConfig) AddRemote(name string, host string, port string) error {

	remotes := &RemoteConfigData.Remotes
	if _, exists := (*remotes)[name]; exists {
```

File: cmd/cmd/serverAdd.go (hunk 1 above)
Comment (tharinduwijewardane) [reply cmd/cmd/serverAdd.go]: As discussed offline,
It is not a duplication. Common code which can be re-used is already moved to the RemoteConfigData.

File: cmd/utils/remoteConfig.go (hunk 2 above)
Comment (tharinduwijewardane) [reply cmd/utils/remoteConfig.go]: As discussed offline,
It is not worth moving two lines to a separate method and calling it.

**Pull request merged**